src/main/java/core/basesyntax/figures/Rectangle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.abstraction.Figure;
+
+public class Rectangle extends Figure {
+
+    protected final int sizeOfTopSide;
+    private final int sizeOfLateralSide;
+
+    public Rectangle(String color, int sizeOfTopSide, int sizeOfLateralSide) {
+        super(color);
+        this.sizeOfTopSide = sizeOfTopSide;
+        this.sizeOfLateralSide = sizeOfLateralSide;
+    }
+
+    @Override
+    public String toString() {
+        return "Figure: rectangle, area: " + getArea() + " sq.units,"

```

Comment:
Let’s make your code consistent. Use constants for figure name in all classes or write it in method ```toString()``` (I prefer the second variant)